addons/l10n_ar_withholding/models/l10n_ar_partner_withholding.py

```diff
@@ -0,0 +1,46 @@
+from odoo import models, fields
+# from odoo.exceptions import ValidationError
+import logging
+# from dateutil.relativedelta import relativedelta
+_logger = logging.getLogger(__name__)
+
+
+class L10nArPartnerTax(models.Model):
+    _name = "l10n_ar.partner.tax"
```

Comment:
Why do we still need this model?

Reply:
@jco-odoo  Because the aliquots of each tax can vary over time, so we need different aliquots to be applied at different moments depending on the payment date. We can talk about this point on the Follow up @vbe-odoo